packages/engine/src/simulation/command.rs

```diff
@@ -82,25 +124,32 @@ impl CreateRemoveCommands {
         Ok(())
     }
 
-    pub fn merge(&mut self, mut other: CreateRemoveCommands) {
-        self.create.append(&mut other.create);
-        self.remove.append(&mut other.remove);
+    pub fn merge(&mut self, mut other: Commands) {
+        self.create_remove
+            .create
+            .append(&mut other.create_remove.create);
+        self.create_remove
+            .remove
+            .append(&mut other.create_remove.remove);
+        if self.stop.is_none() {
+            self.stop = other.stop;
+        }
```

Comment:
How do we want to handle multiple stop message?
- Use the first one (as it is now)
- Use the latest
- Prioritize `"error"` > `"warning"` > `"success"`
- Store a list of stop messages and send them all back to the orchestrator